lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -263,6 +265,26 @@ def destroy_image(action_handler, image_spec, image_options)
       end
     end
 
+    def user_data
+      # TODO: Make this use HTTPS at some point.
+      <<EOD
+<powershell>
+winrm quickconfig -q
+winrm set winrm/config/winrs '@{MaxMemoryPerShellMB="300"}'
+winrm set winrm/config '@{MaxTimeoutms="1800000"}'
+winrm set winrm/config/service '@{AllowUnencrypted="true"}'
+winrm set winrm/config/service/auth '@{Basic="true"}'
```

Comment:
Yeah, btw we should try not to use allowenencrypted + basic -- we should eventually (maybe not in this pr) use negotiate instead. If you use SSL, we can use negotiate rather than basic. See the auth table at https://github.com/chef/knife-windows/blob/master/README.md. Basically, allowunencrypted  + basic is fine for test-kitchen, but not for provisioning real infra.To configure SSL, you'll need to create a cert on the system running provisioning and then send it to the remote system and configure a listener.
MaxMemoryPerShellMB should be a lot higher -- in fact I think the default is 1G on Server 2012 and later. I'm assuming